dispatcher/messages.py

```
@@ -0,0 +1,225 @@
+
+    if not values:
+        raise Exception("No update values provided in the message.")
+
+    if not selector:
+        raise Exception("No selector provided.")
+
+    return selector, values
+
+
+def get_object_ids(
+    selector: dict, resource_endpoint: str, selector_mapping: dict = {}
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)